packages/server/src/cli.ts

```diff
@@ -310,8 +310,12 @@ const bold = (text: string | number): string | number => {
 	} else {
 		logger.warn("Launched without authentication.");
 	}
+	if (options.disableTelemetry) {
+		logger.info("Telemetry is disabled");
+	}
 
-	const url = `http://localhost:${options.port}/`;
+	let schema = options.allowHttp ? "http" : "https";
```

Comment:
This can be a `const`.

Reply:
Also I think it might be called `scheme` rather than `schema`?

Reply:
I had it as protocol to begin with. I will change it backsies.